website/content/docs/release-notes/v0_17_0.mdx

```diff
@@ -0,0 +1,129 @@
+---
+layout: docs
+page_title: v0.17.0
+description: |-
+  Boundary release notes for v0.17.0
+---
+
+# Boundary 0.17.0 release notes
+
+**GA date:** July 31, 2024
+
+@include 'release-notes/intro.mdx'
+
+## New features
+
+<table>
+  <thead>
+    <tr>
+      <th style={{verticalAlign: 'middle'}}>Feature</th>
+      <th style={{verticalAlign: 'middle'}}>Update</th>
+      <th style={{verticalAlign: 'middle'}}>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Centralized tag management for workers
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      GA
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+     Prior to this version, if you wanted to edit or update worker tags, you had to do it using the worker configuration file, the CLI, or the API. Now, you can edit worker tags directly in the Boundary UI.
+      <br /><br />
+      Learn more:&nbsp;<a href="/boundary/docs/concepts/filtering/worker-tags">Worker tags</a>.
+    </td>
+  </tr>
+
+   <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Multi-scope roles and inheritance
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      GA
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      You can now assign a single role to multiple scopes, making it easier to grant permissions to users who must access resources across multiple scopes. You can also configure children scopes to inherit roles.
+      <br /><br />
+      Learn more:&nbsp;<a href="/boundary/docs/concepts/security/permissions">Permissions in Boundary</a>.
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+      Improved worker failure handling
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      GA
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+      When you attempt to connect to a target, Boundary randomly selects a worker that has the matching tags to proxy the connection. Before release 0.17.0, unhealthy workers that had issues related to the external storage provider were eligible to proxy connections. The connections would fail, and users had to restart the connection until Boundary selected a healthy worker. <br /><br />
+      In this release, Boundary removes workers from the connection pool if they have issues with the external storage provider for improved worker failure handling.
```

Comment:
One thing to validate is if connection pool is the right terminology, otherwise this is solid

Reply:
Good catch, thanks! I think this was just an internal reference, so I rephrased it to be more generic:

> Boundary removes workers from _the pool of available workers_ if they have issues with the external storage provider